stablehlo/dialect/Base.cpp

```diff
@@ -75,24 +75,22 @@ bool isCompatibleForHloTypeInference(Type tp1, Type tp2) {
   // ops with partially inferred types to pass verification.
   auto stp1 = tp1.dyn_cast<ShapedType>();
   auto stp2 = tp2.dyn_cast<ShapedType>();
-  if (stp1 && stp2) {
+  if (stp1 && stp2)
     return succeeded(verifyCompatibleShapeWithBounds(stp1, stp2)) &&
            isCompatibleForHloTypeInference(stp1.getElementType(),
                                            stp2.getElementType());
-  }
 
   // Quantization: In the most general case, we allow any combination of
   // quantized/non-quantized across any combination of operands/results,
   // and some differences in quantization parameters across operands/results.
   // Individual ops may introduce additional constraints.
   auto qtp1 = tp1.dyn_cast<quant::QuantizedType>();
   auto qtp2 = tp2.dyn_cast<quant::QuantizedType>();
-  if (qtp1 && qtp2) {
+  if (qtp1 && qtp2)
```

Comment:
LLVM style guide:

```
// Use braces on the outer `if` to avoid a potential dangling `else`
// situation.
if (isa<VarDecl>(D)) {
  if (shouldProcessAttr(A))
    handleAttr(A);
}
```